bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -3148,7 +3147,8 @@ ReplicaImp::ReplicaImp(bool firstTime,
                                 config_.replicaPrivateKey,
                                 config_.publicKeysOfReplicas);
     repsInfo = new ReplicasInfo(config_, dynamicCollectorForPartialProofs, dynamicCollectorForExecutionProofs);
-    viewsManager = new ViewsManager(repsInfo, sigManager, config_.thresholdVerifierForSlowPathCommit);
+    viewsManager =
+        new ViewsManager(repsInfo, sigManager, CryptoManager::instance().thresholdVerifierForSlowPathCommit());
```

Comment:
I guess this one caches the pointer and will need to implement the interface for key exchange registration

Reply:
This is a good catch! I'll change it not to pass it as an argument.